task/tasks.json

```diff
@@ -235,6 +235,29 @@
       "WaitGroup"
     ]
   },
+  "ObjectStat": {
+    "type": "required",
+    "path": "object_stat",
+    "description": "stat a remote object by request headObject",
+    "inherited_value": [
+      "Format",
+      "Key",
+      "ObjectMeta",
```

Comment:
`ObjectMeta` is a runtime value for `ObjectStat`, because it will be updated while runtime.